hack/telepresence-setup.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Script for setting up telepresence to run the KServe controller
+# locally, fully connected to the cluster (it works also for the
+# incoming direction when used as a webhook)
+#
+# Usage:
+#        # Install everything. This command is idempotent and can
+#        # be called multiple times
+#        ./telepresence-setup.sh
+#
+#        # Remove everything from telepresence
+#        ./telepresence-setup.sh uninstall
+#
+# More information for using telepresence for developing:
+#
+# * https://codefresh.io/blog/telepresence-2-local-development/
+# * https://ttt.io/kubernetes-admission-controller
+
+set -o pipefail
+set +e
+
+if [[ "$1" == "uninstall" ]]; then
+  telepresence helm uninstall
+  kubectl delete ns ambassador
+  telepresence quit -s
+  [ -d "${TMPDIR}/k8s-webhook-server" ] && rm -rf "${TMPDIR}/k8s-webhook-server"
+  exit 0
+fi
+
+if ! type telepresence >/dev/null 2>&1; then
+  echo "Telepresence is not installed."
+  echo "Please install it, e.g. with \"brew install telepresence\""
```

Comment:
perhaps pointing to here: https://www.getambassador.io/docs/telepresence/latest/install

Since `brew` is a Mac specific tool?

Reply:
Good point, will update. Thanks!